### DISCUSSION TITLE
Automatic change of color scheme

### DISCUSSION THREAD

**Discussion by aKaiser79** [javikr/sofatime#25] [Ideas]
The following consideration for the automatic change of the color scheme.  Wouldn't it be better if the color scheme was displayed correctly when the app was opened?  Currently I open the app, it shows the last color scheme, then the display dims and the new color scheme appears.  It would be better if the app started immediately with the right color scheme.

**Comment by javikr** [comment]
Yes, I’m agree. The dim effect is displayed to avoid a flash efect when reloading screens. At launch time maybe I can improve this behaviour to avoid the dim/flash effect and show the app with the correct theme. 

As this is a feature improvement, I will transform this idea to an issue.

Thanks!
